sdk/js/index.ts

```diff
@@ -0,0 +1,15 @@
+import { getContract, Address, GetContractReturnType, PublicClient, WalletClient, Abi } from "viem";
+import StreamManagerContractType from "../../.build/StreamManager.json";
```

Comment:
Are we using webpack at all? If we ever do, we can add path resolves so we can do:

```
import StreamManagerContractType from "@build/StreamManager.json";
```

which makes managing relative imports a lot easier.